examples/content_reuse/content_reuse_mapping.yaml

```diff
@@ -16,15 +15,15 @@ graph:
   - node_name: data_preparation
     inputs: [path_to_result_folder]
     outputs: [path_to_cosine_similarity_matrix, path_to_complete_data_set]
-    operation: contentreuse.ScoringDataPreparation
+    operation: contentreuse.scoring_data_preparation
```

Comment:
Shouldn't the operation home be also in underline, `content_reuse`?

Reply:
Should I change it?